discord/message.py

```diff
@@ -452,6 +455,87 @@ def guild_id(self) -> Optional[int]:
         return self._parent.guild_id
 
 
+class MessageSnapshot:
+    """Represents a message snapshot attached to a forwarded message.
+
+    Attributes
+    -----------
+    type: :class:`MessageType`
+        The type of the snapshotted message.
+    content: :class:`str`
+        The actual contents of the snapshotted message.
+    embeds: List[:class:`Embed`]
+        A list of embeds the snapshotted message has.
+    attachments: List[:class:`Attachment`]
+        A list of attachments given to the snapshotted message.
+    timestamp: :class:`datetime.datetime`
```

Comment:
```suggestion
    created_at: :class:`datetime.datetime`
```

Discord.py changes these attributes to saner names, e.g. `created_at`/`edited_at`.